src/routes/home/components/ProfessionDetails.tsx

```diff
@@ -61,17 +63,16 @@ const ProfessionDetails = ({
         />
         {discipline.leafDisciplines && (
           <NumberText>
-            {moduleAlreadyStarted && `${Math.floor(progress * discipline.leafDisciplines.length)}/`}
-            {discipline.leafDisciplines.length}
+            {moduleAlreadyStarted && doneModules}/{discipline.leafDisciplines.length}
```

Comment:
Check for `moduleAlreadyStarted` is redundant since it is already checked in line 40 for `doneModules`
```suggestion
            {doneModules}/{discipline.leafDisciplines.length}
```
Edit: Seems like it breaks the behavior since it now just shows `/24 Module` instead of `0/24 Module`

Reply:
couldn't replicate the breaking behavior, including for completely new jobs - is this still an issue?